ansible_bender/utils.py

```diff
@@ -17,6 +17,19 @@
 out_logger = logging.getLogger(OUT_LOGGER)
 
 
+# template playbook dictionary
+playbook_template = {'hosts': 'all',
```

Comment:
I'd actually store this as a string with yaml inside so we can precisly control the formatting -- I'm pretty sure that yaml.safe_dump will make it hard to edit afterwards.